l10n_it_dichiarazione_intento/models/dichiarazione_intento.py

```diff
@@ -15,8 +15,17 @@ class DichiarazioneIntentoYearlyLimit(models.Model):
 
     company_id = fields.Many2one('res.company', string='Company')
     year = fields.Char(required=True)
-    limit_amount = fields.Float()
-    used_amount = fields.Float(compute='_compute_used_amount')
+    limit_amount = fields.Float(
+        string='Plafond'
+    )
+    used_amount = fields.Float(
+        string='Issued Declarations',
```

Comment:
La label di questo campo non mi sembra coerente con il suo nome. Puoi verificare?

Reply:
verifico, grazie

Reply:
ok, ho rifatto il punto della situazione :) le lable sono state cambiate in base a quanto richiesto dal funzionale. non ho modificato il nome del campo per evitare la creazione di script di migrazione.

Reply:
ok, se in questa fase non hai tempo di fare lo script potresti almeno aggiungere un TODO com'è stato fatto [qui](https://github.com/OCA/l10n-italy/blob/3b7edc52d92dc05e8203fc5260f5558ad4ef1d71/l10n_it_ddt/models/stock_picking_package_preparation.py#L118)?

Reply:
ok